src/Parser/Warrior/Arms/Modules/Core/ColossusSmash.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import { formatPercentage } from 'common/format';
+import SpellLink from 'common/SpellLink';
+import SPELLS from 'common/SPELLS';
+import Wrapper from 'common/Wrapper';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import SpellUsable from '../Features/SpellUsable';
+
+class ColossusSmashAnalyzer extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    spellUsable: SpellUsable,
```

Comment:
Err, I spoke a little soon.  You should remove spellUsable since you don't use it here.